misk/src/main/kotlin/misk/cloud/aws/environment/AwsInstanceMetadataModule.kt

```diff
@@ -26,7 +27,7 @@ class AwsInstanceMetadataModule : AbstractModule() {
         val httpClient = OkHttpClient()
         val request = Request.Builder()
                 .get()
-                .url("$AWS_INSTANCE_METADATA_URL/$metadataCategory")
+                .url("${AWS_INSTANCE_METADATA_URL}/$metadataCategory")
```

Comment:
nit: Add curly braces to `metadataCategory` as well.

Reply:
or none at all. They only should be necessary when the expression is not an identifer

Reply:
no reason to mandate extra syntax that we don’t need